vaadin-renderer-flow-parent/vaadin-renderer-flow-integration-tests/src/main/java/com/vaadin/flow/data/renderer/tests/LitRendererPage.java

```diff
@@ -87,6 +87,13 @@ public LitRendererPage() {
         toggleAttachedButton.setId("toggleAttachedButton");
         add(toggleAttachedButton);
 
+        NativeButton setCheckboxRenderer = new NativeButton("Set renderer with checkbox", e -> {
```

Comment:
Nit: Would it make sense to place this button after the "Set simple LitRenderer" button instead?

Reply:
Done.